content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -14,70 +14,116 @@ OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
 The following example demonstrates automatic and manual instrumentation via an
 ASP.NET Core app.
 
-First, install all the required packages:
+First, create your basic ASP.NET Core site:
+
+```shell
+dotnet new mvc
+```
+
+Next, Add the Core OpenTelemetry packages
 
 ```shell
 dotnet add package OpenTelemetry.Exporter.Console
-dotnet add package OpenTelemetry.Extensions.Hosting --prerelease
-dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
-dotnet add package OpenTelemetry.Instrumentation.Http --prerelease
-dotnet add package OpenTelemetry.Instrumentation.SqlClient --prerelease
+dotnet add package OpenTelemetry.Extensions.Hosting
 ```
 
-Note that the `--prerelease` flag is required for all instrumentation packages
-because they are all are pre-release.
+Now let's add the automatic instrumentation packages for ASP.NET Core. This will
```

Comment:
Can't recollect what was the end result of the debate about the terminology, but OTel .NET repo uses the term "instrumentation library", and "automatic instrumentation" is used only for the auto-instrumentation project.

Reply:
The terms aren't defined so I went for what would make sense to someone just "getting started" rather than being specific about the terminology at this stage. If there's definitions for these laid out somewhere then I'll happily use them though.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#automatic-instrumentation
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library

Reply:
Thanks, I hadn't found those and noone else was able to point me to them. I'll update the whole PR based on those (and also some talk slides :D).

Reply:
Yeah, we use that terminology here too (even though I disagree with it 🙂)